python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -518,6 +518,20 @@ def any_deps_in_progress() -> "AnyDepsCondition":
                 "any_deps_in_progress"
             )
 
+    @experimental
+    @staticmethod
+    def all_deps_blocking_checks_passed() -> "AllDepsCondition":
+        """Returns an AutomationCondition that is true for any partition where all upstream
+        blocking checks have passed, or will be requested.
+        """
```

Comment:
Why is "will be requested" included here where the top-level language is "passed"?

The docblock should explain this.